Scripts/script-DBotPreparePhishingData.yml

```diff
@@ -0,0 +1,219 @@
+commonfields:
+  id: DBotPreparePhishingData
+  version: -1
+name: DBotPreparePhishingData
+script: |+
+  import base64, uuid, re, zlib
+  import demisto_ml
+
+  TRAIN_FILE_NAME = str(uuid.uuid4()) + ".train"
+  ALL_LABELS = "*"
+  TAG_PREFIX = demisto_ml.DEFAULT_LABEL_PREFIX
+
+  def get_context(incident_id):
+      res = demisto.executeCommand("getContext", {'id': incident_id});
+      try:
+        return res[0]['Contents']['context']
+      except:
+        return {}
+
+  def canonize_label(label):
+      return label.lower().strip().replace(" ","_")
+
+  def get_phishing_map_labels(comma_values):
+      if comma_values == ALL_LABELS:
+          return comma_values
+      comma_values = comma_values.lower()
+      values = comma_values.split(",")
+      labels_map = {}
+      for v in values:
+          v = canonize_label(v)
+          if ":" in v:
+              splited = v.split(":")
+              labels_map[splited[0].strip()] = canonize_label(splited[1])
+          else:
+              labels_map[v] = canonize_label(v)
+      return labels_map
+
+  def get_incidents(query, size, is_context_data_needed):
+      res = demisto.executeCommand("getIncidents", {"query": query, "size": size, "sort": "created"})
+      if res[0]['Type'] == entryTypes['error']:
+          return_error(str(res[0]['Contents']))
+      incident_list = res[0]['Contents']['data'] or []
+
+      for i in incident_list:
+          custom_fields = i.get('CustomFields', {}) or {}
```

Comment:
Please add a comment here to explain that this is done to flatten the custom fields in the incident